mesh/tetrahedron.hpp

```diff
@@ -76,7 +76,9 @@ class Tetrahedron : public Element
    /** Reorder the vertices so that the longest edge is from vertex 0
        to vertex 1. If called it should be once from the mesh constructor,
        because the order may be used later for setting the edges. **/
```

Comment:
Extend the comment to say that length is defined by a metric based on geometric length and, in the case of a tie, global index?

Reply:
See [6347978](https://github.com/mfem/mfem/pull/3847/commits/6347978eb2b59f1056317f86a1edc197ebf1e7bf).